beacon-chain/rpc/beacon_chain_server.go

```diff
@@ -72,3 +73,39 @@ func (bs *BeaconChainServer) ListValidatorBalances(
 	}
 	return &ethpb.ValidatorBalances{Balances: res}, nil
 }
+
+// GetValidators retrieves the active validators with an optional historical epoch flag to
+// to retrieve validator set in time.
+//
+// TODO(#3045): Implement validator set for a specific epoch. Current implementation returns latest set,
+// this is blocked by DB refactor.
+func (bs *BeaconChainServer) GetValidators(
+	ctx context.Context,
+	req *ethpb.GetValidatorsRequest) (*ethpb.Validators, error) {
+
+	validators, err := bs.beaconDB.Validators(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not retrieve validators: %v", err)
+	}
+
+	pageSize := int(req.PageSize)
+
+	start, err := strconv.Atoi(req.PageToken)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not get page token: %v", err)
```

Comment:
Isn't page token optional? 

If not, this isn't an internal error, this is an invalid argument: https://github.com/grpc/grpc-go/blob/977142214c45640483838b8672a43c46f89f90cb/codes/codes.go#L49